assets/cohere.yaml

```diff
@@ -145,11 +145,81 @@
       support at cohere.ai.
 
 - type: model
-  name: Cohere Representation model
+  name: Cohere Command
   # General
   organization: Cohere
   description: >
-    The Representation model is a language model trained by Cohere for
+    This model is a generative model optimized to follow commands in the prompt.
+  dependencies: [Cohere Base, coheretext]
+  created_date:
+    value: 2021-11-15
```

Comment:
Is this the correct date for Cohere Command? My impression is it was a more recent model. 

For example, should it be 2022-11-08 to match the version Command is currently pinned to (https://docs.cohere.com/docs/command-beta)?

Reply:
Updated to Jan 2023